src/webui/www/private/upload.html

```diff
@@ -8,6 +8,15 @@
     <link rel="stylesheet" href="css/Window.css?v=${CACHEID}" type="text/css" />
     <script src="scripts/lib/MooTools-Core-1.6.0-compat-compressed.js"></script>
     <script src="scripts/download.js?v=${CACHEID}"></script>
+    <script>
+        'use strict';
+
+        window.addEvent('domready', function() {
```

Comment:
You should move it to the `<script>` in `<body>` which is located below.
And possibly strip `window.addEvent('domready', function() {` since it is run after loading the DOM.

Reply:
done